python/ray/data/_internal/execution/interfaces/physical_operator.py

```diff
@@ -179,6 +180,8 @@ def get_next(self):
     execution (now the default execution mode) the calls could be interleaved.
     """
 
```

Comment:
Could you add a comment that this referring to labels of remote tasks? I don't think it's entirely obvious which label this refers to from the name